lib/UA.js

```diff
@@ -256,6 +259,39 @@ module.exports = class UA extends EventEmitter
 
     return message;
   }
+  
+  /**  
+   * Arbitrary client transaction 
+   */
+  sendRequest(method, target, params, headers, body, handlers, credential)
+  {
+    debug('sendRequest()');
+    const request = new SIPMessage.OutgoingRequest(method, this.normalizeTarget(target), 
+      this, params, headers, body);
+    const requestSender = new RequestSender(this, request, handlers, credential);
+
+    requestSender.send();
+  }
+
+  /** 
+   * Create subscriber
+   */
+  subscriber(target, options)
```

Comment:
Let's use verbs for methods for consistency: `subscribe()`

Reply:
OK